packages/tldraw/src/state/data/migrate.ts

```diff
@@ -126,13 +126,13 @@ export function migrate(state: TDSnapshot, newVersion: number): TDSnapshot {
     })
   })
 
-  if (version < 15.4) {
-    settings.dockPosition = 'bottom'
-  }
+  // if (version < 15.4) {
+  //   settings.dockPosition = 'bottom'
+  // }
 
-  if (version < 15.5) {
-    settings.exportBackground = TDExportBackground.Transparent
-  }
+  // if (version < 15.5) {
+  //   settings.exportBackground = TDExportBackground.Transparent
+  // }
```

Comment:
I think we can safely delete all of this. Migration doesn't need to care about settings anymore.

Though actually, would we need to _delete_ `settings` from something?